opt/Makefile

```diff
@@ -142,42 +142,44 @@ endif
 
 #----------------------------------------------------------------------------------------------
 
+TEST_CMD=DEVICE=$(DEVICE) PYDEBUG=$(PYDEBUG) python3 -m RLTest $(TEST_ARGS) --module $(INSTALL_DIR)/redisai.so
+
 TEST_REPORT_DIR ?= $(PWD)
 ifeq ($(VERBOSE),1)
 TEST_ARGS += -v
 endif
 ifeq ($(TEST),)
-TEST=basic_tests.py
+TEST=tests_common.py tests_pytorch.py tests_onnx.py tests_tflite.py tests_tensorflow.py
```

Comment:
The $(TEST) variable is there for picking a single test for debugging purposes (in a file.py:specific_test form). For execution of several tests, I bet RLTest can run all tests in a given dir.

Reply:
@rafie does this alter the behaviour?
```
make -C opt test TEST="tests_common.py:test_set_tensor  tests_pytorch.py:test_run_torch_model"
```
would still be valid right? ( testing it )

Reply:
Well, from a second look, my design seems to be wrong here. I meant (as it is in other modules), that TEST would be given something that would match the RLTEST --test argument, and then RLTest be invoked with -s (for output from Redis) and PYDEBUG set for python debugging. Instead, the Makefile would override TEST with basic_test.py (which was the only test file). I think it's better to let the user explicitly specify tests (like you do in your example), and not pass --tests $(TEST) to RLTest at all if TEST is not set. Something like:
```
ifneq ($(TEST),)
TEST_ARGS += --test $(TEST)
emdif
```